kobo/cli.py

```diff
@@ -215,6 +216,9 @@ def __init__(self,
                 ["--password", "specify password"]
             )
 
+        if add_hub_option:
+            self._add_opts(["--hub", "specify hub"])
```

Comment:
It seems that the help string should specify that this is a URL, it doesn't seem self-evident.

Reply:
Good idea.  Addressed in 3fcfebc9175a3096a7b557707c2d9990d06a3834.